locations/spiders/sunbelt_rentals_us_ca.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+
+import scrapy
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class SunbeltRentalsUsCaSpider(scrapy.Spider):
+    name = "sunbelt_rentals_us_ca"
+    item_attributes = {"brand": "Sunbelt Rentals", "brand_wikidata": "Q102396721"}
+    allowed_domains = ["sunbeltrentals.com"]
+
+    def start_requests(self):
+        yield scrapy.Request(
+            "https://api.sunbeltrentals.com/web/api/v1/locations?latitude=39.0171368&longitude=-94.5985613&distance=5000",
+            headers={
+                "Client_id": "0oa56ipgl8SAfB1kE5d7",
+                "Client_secret": "6yNzPOIbav3xJ0XMFRI9cCKjEmqcKXiPVPhQS7eo",
+                "Logintoken": "",
+                "Request-Id": "12323sunbelt",
+                "X-Correlation-Id": "qjElWEoUHtD5oK0uqMbCivbwL632fc23bef67923b234f8c19",
+                "Origin": "https://www.sunbeltrentals.com",
+                "Referer": "https://www.sunbeltrentals.com/",
+                "Sec-Ch-Ua-Platform": "macOS",
+                "Authorization": "Bearer",
+                "Channel": "WEBAPP",
+                "Companyid": 0,
+                "Host": "api.sunbeltrentals.com",
+            },
```

Comment:
This can be significantly simplified by removing unnecessary headers. Perhaps try:

```
headers = {
  "client_secret": "6yNzPOIbav3xJ0XMFRI9cCKjEmqcKXiPVPhQS7eo",
  "client_id": "0oa56ipgl8SAfB1kE5d7",
  "companyId": "0",
}
yield JsonRequest(url="https://api.sunbeltrentals.com/web/api/v1/locations?latitude=0&longitude=0&distance=100000", headers=headers)
```

Those are the minimal URL parameters and headers needed for the API to return all locations. The Client ID and Client Secret appear to be the same for everyone (not session dependent). Omitting URL parameters will result in only summary information being returned for each location.